tests/e2e/test_client.py

```diff
@@ -300,6 +301,27 @@ def test_insert_table_json_success(self):
                 InsertTableResult(rows_written=1),
             )
 
+    @unittest.skip("This is a plus subscription endpoint.")
```

Comment:
Is it ?

Reply:
You're right, I'm not sure why all of these have it, will reword